server/protocols/jmap/src/test/java/org/apache/james/jmap/utils/JsoupHtmlTextExtractorTest.java

```diff
@@ -71,6 +71,64 @@ public void toPlainTextShouldUpperCaseLabels() {
     }
 
     @Test
+    public void toPlainTextShouldHandleListsWell() {
+        String html = "<ul>Here is my awesome list:" +
+            "  <li>JMAP</li>" +
+            "  <li>IMAP</li>" +
+            "</ul>" +
+            "<p>Followed with some text</p>" +
+            "<p>And some other text</p>";
+        String expectedPlainText = "Here is my awesome list:  \n" +
+            " - JMAP  \n" +
+            " - IMAP\n" +
+            "\n" +
+            "Followed with some text\n" +
+            "\n" +
+            "And some other text\n" +
+            "\n";
+        assertThat(textExtractor.toPlainText(html)).isEqualTo(expectedPlainText);
+    }
+
+    @Test
+    public void tableShouldBeWellHandled() {
+        String html = " <table style=\"width:100%\">\n" +
+            "  <tr>\n" +
```

Comment:
I've just added a testing on Tika and it makes sense it converts `<th>` and `<td>` to "\t".
As we knew that textHtml should be used as quote when reply email on mobile so it would be very happy with that convert for table. Sorry if it requires more :(
Here the code I tested:
input:
`<table>
        <tr>
            <th>Caption 1</th>
            <th>Caption 2</th>
        </tr>
        <tr>
            <td>Value 1</td>
            <td>Value 2</td>
        </tr>
    </table>`
Output: 
`"\n\tCaption 1\tCaption 2\n\tValue 1\tValue 2\n\n`